book/website/community-handbook/local-build.md

```diff
@@ -0,0 +1,67 @@
+5. Clone the repo: `git clone https://github.com/alan-turing-institute/the-turing-way`
+6. Change into the repo: `cd the-turing-way`
+7. Change into the correct folder `cd book/website`
+8. Install the dependencies _into your conda environment_
+   ```
+   pip install -r requirements.txt
+   ```
+9. And now build the book:
+   ```
+   jupyter-book build .
+   ```
+10. The output of the build process will provide output such as below that demonstrate how you can view the book locally:
+    ```
+    ===============================================================================
+
+    Finished generating HTML for book.
+    Your book's HTML pages are here:
+        _build/html/
+    You can look at your book by opening this file in a browser:
+        _build/html/index.html
+    Or paste this line directly into your browser bar:
+        file:///Users/sgibson/source/github/alan-turing-institute/the-turing-way/book/website/_build/html/index.html
+
+    ===============================================================================
+    ```
+    
+### Build the book while working on a Pull Request
+If you would like to preview a version of the book from a certain branch (perhaps to render the book while working on a PR) then simply switch to the required branch and rebuild the book as in step 10:
+   ```
+   git checkout mybranch
+   jupyter-book build .
+   ```
+Follow the link as before and you will see changes specific to that branch rendered.
+
+## Why did we recommend using miniconda?
+- Running the `juypter-book` command to build the Turing Way book relies on having a python installation on your machine. As with any other programming language such as R or Julia, any given python installation might look different from another due to the packages or libraries that come with the installation. 
+- In particular, some packages depend on the presence of specific versions of other packages to function, and so to ensure your local build works smoothly you will want to minimize as much mismatched dependencies as possible.
+- But this can be difficult! Even with an organized, concerted effort, package management for programming languages naturally throws up dependency issues. Python packages, for reasons not discussed here, tend to suffer from dependency issues a bit more than other languages (note that all languages do!) and one guaranteed way to come across such an issue by trying to maintain all of your python projects using just one, large set of packages, each at a specific version. You simply cant cater to the needs of all package dependencies this way. https://xkcd.com/1987/   
+![](https://imgs.xkcd.com/comics/python_environment.png)
+- Anaconda is package manager that can install Python, R and Julia distributions. *Miniconda* is just a smaller package manager that by default installs a smaller set of packages which means in terms of dependencies there are less things to go wrong.
```

Comment:
```suggestion
```
i will suggest adding the links to the Anaconda and miniconda documentation or site.